bool[] ping
bool[] torque_enable
# string[] drive_mode
string[] operating_mode